src/logic/proposals/hooks/get-proposals.ts

```diff
@@ -28,9 +28,7 @@ export const getProposals = async (
     };
   });
 
-  // TODO: load this just once for all proposals and save to state
-  const votingTime = await api3Voting.voteTime();
-
+  const votingTime = EPOCH_LENGTH;
```

Comment:
Voting time is not changeable and is equal to `EPOCH_LENGTH` so it's fine to use it as constant.